keys.js

```diff
@@ -0,0 +1,26 @@
+/*
+	FILE: Stores Keys for various popular websites
+	Current:  Spotify, Twitter and OMDb
+*/
+
+var twitterKeys = {
+	consumer_key:'JFke8Oq9QN7pro9OAmTkcGwZZ',
+	consumer_secret: 'tzVsC6cUy6dvM2tqBPbfREArJF6bROmHaH8hkaTbfuxLbhsSLI',
+	access_token_key: '273098233-JuMKJT4W2l8kkIozzI2UNYQzBjUYqFEil0AjKDXS',
+	access_token_secret: 'T7Uv2UICNwIanJpfk833OE4KPsZQBg605RllzqnATscU4'
+}
+
+var spotifyKeys = {
+	client_id:'e90b297605b34231a25fb8d0852e41c0',
+	client_secret: '19782778d5e946edb372e86bd2fa10c9'
+}
+
+var oMDbAccessKey = {
+	accessKey: '40e9cece'
+}
+
+module.exports = { 
+	spotifyKeys: spotifyKeys,
+	twitterKeys: twitterKeys,
+	oMDbAccessKey: oMDbAccessKey
```

Comment:
I would create the instances of your api's instead of making them in your liri file.